docs/custom-buildpacks.md

```diff
@@ -0,0 +1,285 @@
+# Buildpacks
+Korifi utilizes [kpack](https://github.com/buildpacks-community/kpack) to build Container Images combining [Cloud Native Buildpacks (CNB)](https://buildpacks.io) with application source code.
+
+## Default Buildpacks
+
+If you choose Helm to deploy Korifi some default buildpacks are pre-configured for you.
+You can see the available buildpacks via the CF CLI using the command 
+
+```bash
+cf buildpacks
+Getting buildpacks as cf-admin...
+
+position   name                            stack                        enabled   locked   filename
+1          paketo-buildpacks/java          io.buildpacks.stacks.jammy   true      false    paketo-buildpacks/java@17.0.0
+2          paketo-buildpacks/go            io.buildpacks.stacks.jammy   true      false    paketo-buildpacks/go@4.12.2
+3          paketo-buildpacks/nodejs        io.buildpacks.stacks.jammy   true      false    paketo-buildpacks/nodejs@4.2.1
+4          paketo-buildpacks/ruby          io.buildpacks.stacks.jammy   true      false    paketo-buildpacks/ruby@0.47.6
+5          paketo-buildpacks/procfile      io.buildpacks.stacks.jammy   true      false    paketo-buildpacks/procfile@5.10.0
+```
+
+Those default buildpacks are configured within the `ClusterStore` and `ClusterBuilder` of the Helm Chart, which can be found [here](../helm/korifi/kpack-image-builder/cluster-builder.yaml)
+ 
+## Custom Buildpacks
+
+If you want to make more buildpacks than the default ones available you have two possibilities, you can either adjust the `ClusterStore` and `ClusterBuilder` within your cluster to include further buildpacks or you provide your own `ClusterBuilder` and use it instead.
```

Comment:
suggestion: ...ones available, you have two possibilities. You can either adjust...